app/src/components/MUIModal/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import MUIModal from '../MUIModal'
+import { styles } from './styles'
+import Button from '@material-ui/core/Button'
+import { withStyles, WithStyles } from '@material-ui/core/'
+
+interface ConfirmationModalProps extends WithStyles<typeof styles> {
+    confirmationText: string
+    handleClose: () => void
```

Comment:
We can scrap this handleClose() from the top-level props interface for this component. This is a lower-level concern to be handled internally by this component. We will discuss whether we want the user to be able to click outside of the modal to close without explicitly providing a response, but if we _do_ want to support that, onCancel will suffice to pass as a callback to MuiModal's handleClose prop